docs/src/Link.doc.js

```diff
@@ -126,8 +137,9 @@ card(
 
 card(
   <Example
+    id="accessibility"
     description={`
-    When providing the content for the link, avoid phrases like "click here" or "go to".
+    When providing the content for the link, avoid phrases like "click here" or "go to" or use accessibilityLabel.
```

Comment:
@AlbertCarreras can we remove the `or use accessibilityLabel`? We do want them to use `accessibilityLabel` when it's required

Reply:
DONE